webhooks/fail2ban/fail2ban/README.md

```diff
@@ -0,0 +1,46 @@
+Fail2Ban
+========
```

Comment:
Everything in `webhooks/fail2ban/fail2ban` should go in the [`integrations` directory](https://github.com/alerta/alerta-contrib/tree/master/integrations) ie. in `integrations/fail2ban`.

Reply:
Moved `webhooks/fail2ban/fail2ban` in `integrations` directory.